dart_sp/11.Dart_medio/2_La_terminal.md

```diff
@@ -35,7 +35,7 @@ Buscaron!? Lo que hace la `-R` es hacer recursiva la eliminación ya se trata de
 
 Por otro lado, van a ver cuando empiecen a investigar un poco, las palabras 'su' y 'sudo'. No se los dije antes, pero __cada sistema operativo funciona con distintos usuarios los cuales cuentan cada uno con diferentes roles__, lo que nos permite evitar que __alguien__ que no debe __haga algo que no debe__.
 
-__El usuario más poderoso se llama `root`__ y es quien tiene todos los permisos. Si ustedes teclean `su`, les va a pedir el _password_ para loguearse como dicho usuario. En Mac por ejemplo, dicho usuario está desactivado por defecto. Por otro lado, si uno teclea `sudo` (_Super User DO_), dota al usuario actual de permisos de administrador.
+__El usuario más poderoso se llama `root`__ y es quien tiene todos los permisos. Si ustedes teclean `su`, les va a pedir el _password_ para loguearse como dicho usuario. En Mac por ejemplo, dicho usuario está desactivado por defecto. Por otro lado, si uno teclea `sudo` (_Super User DO_), dota al usuario actual de permisos de administrador. Cuidado! no existe el comando `su`o `sudo`en windows, el mas parecido es `runas`.
```

Comment:
add a small clarification that in windows it is not like this

Reply:
> 

I have never used windows for programming... but if "runas" is mentioned, I think it is also necessary to clarify that it is not needed in this specific case. I mean "el más parecido es "runas" (aunque no aplica en este caso)." On the other hand, "más" takes tílde.

Reply:
I understand and agree, the thing is that on the other hand I've never used linux so I do not know if this was needed xD. It was just a comment cause when I was translating I was also trying things and su or sudo never work for me and it was exhausting. The same hapened whith other commands.

maybe something erase that, and at the end after all the other commnads add a comment that many of those commnads do not work with windows.

I`ll make another commit